README.md

```diff
@@ -1,6 +1,14 @@
 # Bazel Scala Annex
 
-[![Build Status](https://travis-ci.org/andyscott/rules_scala_annex.svg?branch=master)](https://travis-ci.org/andyscott/rules_scala_annex)
+[![Build Status](https://api.travis-ci.org/higherkindness/rules_scala.svg?branch=master)](https://travis-ci.org/higherkindness/rules_scala)
+
+Bazel scala annex evolved, in part, from the need for Bazel adoption support for large, monorepo Scala projects.
```

Comment:
I'd rather we drop the word annex. Is there something else we can use to distinguish this rule set from the bazelbuild version?

Reply:
Maybe we just call these rules "the scala rules" or "rules_scala" and call the other ones "bazelbuild/rules_scala".

Reply:
We could use `higherkindness/rules_scala` and `bazebuild/rules_scala`; I think using just `rules_scala` might confuse people, especially if they're familiar with the other project